src/utils/dom-tranform-stream.js

```diff
@@ -64,8 +64,9 @@ export class DomTransformStream {
     /** @private {boolean} */
     this.shouldTransfer_ = false;
 
-    /** @const @private */
-    this.vsync_ = Services.vsyncFor(win);
+    const noThrottleFunc = (cb) => Promise.resolve(cb());
```

Comment:
Inline in the line below to avoid extra allocation?

Reply:
I think I tried this and it was invalid syntax ie `opt_func || () => {}` would a if/else be better, or maybe a non-arrow function?

Reply:
I think you can just do `(() => {})`. But if not, the old-style function would definitely work.

Reply:
you are right :) parens made it work, done.